llvm/lib/CodeGen/MachineStableHash.cpp

```diff
@@ -95,13 +95,21 @@ stable_hash llvm::stableHashValue(const MachineOperand &MO) {
   case MachineOperand::MO_Metadata:
     StableHashBailingMetadataUnsupported++;
     return 0;
-  case MachineOperand::MO_GlobalAddress:
-    StableHashBailingGlobalAddress++;
-    return 0;
+  case MachineOperand::MO_GlobalAddress: {
+    const GlobalValue *GV = MO.getGlobal();
+    if (!GV->hasName()) {
+      StableHashBailingGlobalAddress++;
```

Comment:
```suggestion
      ++StableHashBailingGlobalAddress;
```
Prefer preincrement, but you might also want to change the rest of the stats variables.
https://llvm.org/docs/CodingStandards.html#prefer-preincrement

Reply:
Updated the increments of the stats variables.